rest/route/build_task.go

```diff
@@ -94,13 +94,16 @@ func (tbh *tasksByBuildHandler) Run(ctx context.Context) gimlet.Responder {
 	tasks = tasks[:lastIndex]
 	for i := range tasks {
 		taskModel := &model.APITask{}
-		err = taskModel.BuildFromService(&tasks[i])
-		if err != nil {
+
+		if err = taskModel.BuildFromService(&tasks[i]); err != nil {
 			return gimlet.MakeJSONErrorResponder(err)
 		}
 
-		err = taskModel.BuildFromService(tbh.sc.GetURL())
-		if err != nil {
+		if err = taskModel.GetArtifacts(); err != nil {
```

Comment:
Is there a test for the GetArtifacts behavior or this route?

Reply:
it's one of will's original routes so the tests are somewhat arcane.

getartifacts is tested indirectly in the task route.